src/main/java/ee/kertmannik/quiz/client/QuestionRequest.java

```diff
@@ -0,0 +1,34 @@
+package ee.kertmannik.quiz.client;
+
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+
+import java.io.IOException;
+import java.net.SocketTimeoutException;
+import java.net.UnknownHostException;
+
+public class QuestionRequest {
+
+    String url;
+
+    QuestionRequest(String url) {
+        this.url = url;
+    }
+
+    public String getQuestionFromServer(String playerName) throws IOException, QuizException {
+        final OkHttpClient client = new OkHttpClient();
+        final Request request = new Request.Builder()
+                .url(url)
+                .addHeader("x-player-name", playerName)
+                .build();
+        try {
+            final Response response = client.newCall(request).execute();
+            return response.body().string();
```

Comment:
I believe that when response.body() is called, it also need to be closed, and that must happen in finally block. For example, see https://bitbucket.org/fortumo/provider-integration-api/src/866923f9b97d71b629b60dd8c568e95bd2e2262b/src/main/java/io/fortumo/provider/integration/pcaclient/HttpClient.java?at=master&fileviewer=file-view-default doJsonPost method